pallets/chainbridge/src/tests.rs

```diff
@@ -11,6 +11,7 @@ use super::{
 	pallet::Event as PalletEvent,
 	*,
 };
+use frame_benchmarking as _;
```

Comment:
Why is it imported like this? 
If you need somehow import benchmarks then I assume you need to do something like [this](https://github.com/Polkadex-Substrate/Polkadex/blob/4e89260855633dccc78026a2b0644951cdd23549/pallets/asset-handler/src/lib.rs#L25-L26) and not here.